Documentation/policy/language.rst

```diff
@@ -19,9 +19,9 @@ can talk to each other. Layer 3 policies can be specified using two methods:
   destination endpoint is not controlled by Cilium.
 
 * `Entities based`: Entities are used to describe remote peers which can be
-   categorized without knowing their IP addresses. This includes connectivity
-   to the local host serving the endpoints or all connectivity to outside of
-   the cluster. Future versions will allow to define your own entities.
+  categorized without knowing their IP addresses. This includes connectivity
+  to the local host serving the endpoints or all connectivity to outside of
```

Comment:
This seems to forget the 'all' case - https://github.com/cilium/cilium/pull/3541/files#diff-f702af0e53ce258f3754cd310bd6ccbfR210

Reply:
This is not an exhaustive list, all is described in the section that describes this L3 condition.